Shawntru/FitLit-refactor
# Fix/bad data continue

## Is this a fix or a feature?
Feature 

## What is the change?
The posting of data now has conditional added on

## Where should the reviewer start?
fetch.js line 30

## How should this be tested?
Try and press the submit buttons on any of the field without first entering any data.
An alert should pop up informing the user of the missing data. Once all fields are filled the post request will go through.

## Relevant Ticket(s)

@Shawntru 
@adavidson02
@sarah01123
